Clicker/Models/ListDiffableModels/PollDateModel.swift

```diff
@@ -13,11 +13,13 @@ class PollDateModel {
     
     var date: String
     var poll: Poll
+    var index: Int
     let identifier = UUID().uuidString
     
-    init(date: String, poll: Poll) {
+    init(date: String, poll: Poll, index: Int) {
         self.date = date
         self.poll = poll
+        self.index = index
```

Comment:
This is used in `didSelectItemAt` so that when you press on a cell in the vertical view, the polls for the correct date are opened in the horizontal view: `currentIndex` for the data source in the horizontal view is set to this `index`.

Reply:
I don't think having the `index` as a part of the model is that great because the index can change. Instead, inside the section controller's `didSelectItem` function, we could pass back the `PollDateModel` associated with it back to `CardController` to find the `index`. You don't have to add this in this PR though, just letting you know how we would do it.